elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/metric/AggregatedField.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.metadata.metric;
+
+import com.yahoo.elide.datastores.aggregation.metadata.models.Metric;
+
+import lombok.Getter;
+
+/**
+ * A field in a physical/logical table that is actually aggregated by a metric function.
+ * It can be either a metric field from a physical table or a field alias in a subquery.
+ */
+public class AggregatedField {
```

Comment:
Let's remove this concept from the model.  I think it add conceptual weight and not enough value.